File: CT-206/Project02/Project02Sorter.py
#CT-206
#Project02 
#Nicholas Jones
#One subroutine/function which takes in a list of words and returns the same list but in reverse order. 

def reverseList(animals):
    animals.reverse()
    print("\nList has been reversed within function.")
    return animals

animals = ["dog", "cat", "cow", "horse", "bird"]
print ("List of animals: ")
print (animals)

reverseList(animals)
  
print ("\nNew list of animals: ")
print (animals) 

      

File: CT-206/Project02/Project02Sorter.pl
#CT-206
#Project02
#Nicholas Jones
#One subroutine/function which takes in a list of words and returns the same list but in reverse order. 

my @animals = qw(Cat Dog Cow Horse Bird);
print "List of animals: \n";
print join " ", @animals, "\n";

print "\nNew list of animals: \n";
@animals = reverseList();
print join " ", @animals, "\n";

sub reverseList
{
	reverse @animals;
}